source/databricks/calculation-engine/tests/calculation/preparation/transformations/test_charge_types.py

```diff
@@ -12,933 +12,353 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from datetime import datetime, timedelta
+import pytest
+from datetime import datetime
 from decimal import Decimal
 from pyspark.sql import SparkSession, DataFrame
 from pyspark.sql.types import (
-    DecimalType,
-    StringType,
-    StructField,
     StructType,
-    TimestampType,
 )
-from typing import Callable
-from package.calculation.preparation.transformations.charge_types import (
-    _join_with_metering_points,
-    _explode_subscription,
-    _get_charges_based_on_resolution,
+from package.calculation.preparation.transformations import (
     get_tariff_charges,
-    _group_by_time_series_on_metering_point_id_and_resolution_and_sum_quantity,
-    _join_with_grouped_time_series,
-    _get_charges_based_on_charge_type,
+    get_fee_charges,
+    get_subscription_charges,
 )
 from package.codelists import (
     ChargeQuality,
     ChargeType,
     ChargeResolution,
     MeteringPointType,
     SettlementMethod,
-)
-from package.calculation.wholesale.schemas.charges_schema import (
-    charges_master_data_schema,
-)
-from package.calculation.preparation.transformations.charges_reader import (
-    _create_charges_df,
+    MeteringPointResolution,
+    TimeSeriesQuality,
```

Comment:
Idea: How about starting a trend like this?
```suggestion
import package.codelists as E
```
Where `E` would be short for "enum".

Reply:
good idea